src/App.tsx

```diff
@@ -1,14 +1,37 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
+import { Loader } from './components/Loader';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
-import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterBySelect } from './types/FilterBySelect';
+import { filterTodos, getErrorMessage } from './HelperFunctions';
```

Comment:
Rename 'HelperFunctions' to use consistent capitalization, e.g. 'helperFunctions' or 'HELPER_FUNCTIONS' 🔄